src/ai/backend/manager/models/gql_models/image.py

```diff
@@ -464,6 +466,36 @@ class Meta:
         ImagePermissionValueField,
         description=f"Added in 25.3.0. One of {[val.value for val in ImagePermission]}.",
     )
+    installed = graphene.Boolean(
+        description="Added in 25.11.0. Indicates if the image is installed on any Agent."
+    )
+
+    @property
+    def _full_name(self) -> str:
+        return f"{self.registry}/{self.namespace}:{self.tag}"
```

Comment:
@jopemachine Is this correct to guess the full name of an image?

Reply:
To avoid such recurring these questions, I think it would be a good idea to add a canonical field like the one below.
@HyeockJinKim 

```py
    canonical = graphene.String(description="Added in 24.12.0")

    def resolve_canonical(self, info: graphene.ResolveInfo) -> str:
        join = functools.partial(join_non_empty, sep="/")
        return f"{join(self.registry, self.name)}:{self.tag}"
```

Reply:
If we're hesitant to add `canonical` as a field due to the additional overhead it may cause, it could be also good idea to simply add it as a function or property instead.

Reply:
I think I've seen the function that creates canonical in several places, and it seems to be fragmenting?
How about creating a class that takes image registry, name, and tag, and providing canonical as a method?